io/src/pcd_grabber.cpp

```diff
@@ -204,7 +204,7 @@ bool
 pcl::PCDGrabberBase::PCDGrabberImpl::readTARHeader ()
 {
   // Read in the header
-#if WIN32
+#ifdef WIN32
```

Comment:
The author probably meant `_WIN32`

```suggestion
#ifdef _WIN32
```